src/Reflex/TriggerEvent/Base.hs

```diff
@@ -5,6 +5,7 @@
 {-# LANGUAGE GeneralizedNewtypeDeriving #-}
 {-# LANGUAGE MultiParamTypeClasses #-}
 {-# LANGUAGE TypeFamilies #-}
+
```

Comment:
Reasonable, but should be done globally, not just in this one source file.  Also, I think a module comment would serve better.